src/Tools/dotnet-monitor/Exceptions/ExceptionsOperation.cs

```diff
@@ -282,5 +297,10 @@ private static async Task WriteTextExceptionFormat(TextWriter writer, IException
                 await writer.WriteAsync(instance.Message);
             }
         }
+
+        private static bool IncludeActivityId(IExceptionInstance instance)
+        {
+            return !string.IsNullOrEmpty(instance.ActivityId) && instance.ActivityId != Guid.Empty.ToString();
```

Comment:
```suggestion
            return !string.IsNullOrEmpty(instance.ActivityId);
```
I don't think we should check for Guid.Empty, especially since this is not a used sentinel value other than maybe our own tests (which should be changed to use either `null` or `string.Empty`).